### ISSUE TITLE
Does anyone have it working with Safari 9?

### ISSUE THREAD

**Issue by BrianGilbert** [hetima/SafariStand#35]
I'm using El Capitan which now has Safari 9, would really love to use SafariStand, more than happy to help with testing..

**Comment by ssbarnea** [comment]
Interested too, now on Safari 9 yet but I am not willing to try something that doesn't work with current versions of Safari.

**Comment by AdeTheux** [comment]
Working yes, but random crashes when I right click.

**Comment by BrianGilbert** [comment]
@AdeTheux can you explain how you installed it under El Capitan please?

**Comment by AdeTheux** [comment]
Follow these steps: https://github.com/norio-nomura/EasySIMBL/issues/26#issuecomment-117028426

**Comment by BrianGilbert** [comment]
@AdeTheux thanks, I now have a somewhat functional version installed..

**Comment by BrianGilbert** [comment]
![safaristand 2015-09-17 15-40-45](https://cloud.githubusercontent.com/assets/114017/9934569/847ae994-5d52-11e5-9631-fe6513396d53.png)

**Comment by gkarag** [comment]
Using SafariStand Version 8.0.214. Updated to Safari 9 with Mac OS X 10.10.5. Here is the experience for the features I use so far:
1. Working
 - The Stand menu appears and clicking on the items works
 - Show Icon On Sidebar Bookmarks
 - Expand Address Bar Width
2. Not working
 - Show Icon On Tab Bar
 - Suppress Tab Bar Width
3. Crashes
 - Random crashes when I right click on an image (Remove Default Contest Menu Item endabled)

**Comment by demidovakatya** [comment]
+1 to crashes:
* Safari crashes when I right-click on any link.

**Comment by Knorkebrot** [comment]
Do the crashes stop if you disable the context menu extensions in SafariStand? (SafariStand Settings > Context Menu)

**Comment by gkarag** [comment]
@Knorkebrot Nope, Safari still crashes on right click with everything disabled on the Context Menu pane.

**Comment by Knorkebrot** [comment]
@gkarag Too bad, thanks for checking.

**Comment by gkarag** [comment]
Now using SafariStand Version 8.0.215Yosemite on Safari 9 with Mac OS X 10.10.5:

Working:
  * The Stand menu appears and clicking on the items works
  * Show Icon On Sidebar Bookmarks
  * Expand Address Bar Width
  * Suppress Tab Bar Width
  * Remove Default Context Menu Item

Not working (as announced):
  * Show Icon On Tab Bar

Crashes:
 * None so far

**Comment by JayBrown** [comment]
Safari Stand 9.0.215 on OS X 10.10.5 with Safari 9 (10601.1.56.2). No icons on tab bar… not good. Favicon checkbox & text in Stand settings is greyed out. Other settings and checkboxes are greyed out as well, e.g. copy link tag, add target="_blank" etc. So there's reduced functionality with Safari 9, but other than that it seems to work, and it seems to be stable. Don't know yet how it will be with OS X 10.11, but I guess that it's the same, once you get SIMBL to run again.

**Comment by jaspham** [comment]
Works perfectly for me...
